### PR TITLE
Feature/reintroduce missing view

### PR DESCRIPTION
**Pull request by jonsherrard**
#### What have you done
When using default-routes, we have a fallback component again

#### Why have you done it
That was nice functionality, and we figured out a way to get it back

#### Testing carried out to prevent breaking changes
I've added the MissingView test back in

### CODE REVIEW

File: src/shared/default-routes.js [hunk 1]
```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```
Comment (thomasdigby) [inline src/shared/default-routes.js]: I don't quite understand this line, shouldn't it be `MissingView` rather than `Error`?

Comment (jonsherrard) [inline src/shared/default-routes.js]: There was a line `const components.Error || Missing View` above, which I think was for supplying your own Error component from the User Project.

Comment (thomasdigby) [inline src/shared/default-routes.js]: Right, but wouldn't that mean if they passed an `Error` component they'll never see the `MissingView`? Just curious.

Comment (jonsherrard) [inline src/shared/default-routes.js]: Yep they'd not see it anymore. I was assuming this was like `fallback` prop in the previous setup. https://github.com/shortlist-digital/tapestry-wp/blob/develop/src/shared/default-routes.js#L23 I can take it out.

Comment (thomasdigby) [inline src/shared/default-routes.js]: Ah no, it's all good, we could probably do with a little more work in rendering the correct fallback component for production anyway, happy to mergeeeee this

**Pull request merged**